src/Consumer/LoggerConsumer.php

```diff
@@ -15,7 +15,6 @@
 
 use Psr\Log\LoggerInterface;
 use Sonata\NotificationBundle\Exception\InvalidParameterException;
-use Symfony\Component\HttpKernel\Log\LoggerInterface as LegacyLoggerInterface;
```

Comment:
This interface was removed in Symfony `3.0`: https://github.com/symfony/symfony/blob/3.0/UPGRADE-3.0.md#httpkernel